src/components/Graph/Graph.tsx

```diff
@@ -236,29 +203,18 @@ const GraphMemo: FC<GraphMemoProps> = memo(
       >
         {nodeCoordinates !== undefined
           ? Object.keys(nodeCoordinates).map(nodeName => {
-              const docResource = documentation[nodeName] as DocumentationResource;
               const onClickHandler = useCallback(() => {
                 return interactive ? setExpanded(nodeName) : undefined;
               }, [nodeName]);
               return (
                 <Node
                   key={nodeName}
-                  documentation={docResource}
                   ref={(node: HTMLDivElement): void => {
                     nodeRefs.current[nodeName] = node;
                   }}
                   pathwayState={pathway.states[nodeName]}
-                  isOnPatientPath={
-                    evaluatedPathway.pathwayResults
-                      ? getPath(evaluatedPathway.pathwayResults).includes(nodeName) ||
-                        evaluatedPathway.pathwayResults.currentStates.includes(nodeName)
-                      : false
-                  }
-                  isCurrentNode={
-                    evaluatedPathway.pathwayResults
-                      ? evaluatedPathway.pathwayResults.currentStates.includes(nodeName)
-                      : false
-                  }
+                  isOnPatientPath={false}
```

Comment:
We should probably remove this prop all together (or at least rename it if it being used for some styling which we want to keep coupled with this prop)

Reply:
Will remove it. Based on the current designs all nodes should be blue except for the current node being edited, which is red, so I'll implement that here.